CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.6.2 - 2021-01-18
```

Comment:
Not sure if the date is required or if this should just be a starting point for 2.6.2